sdk/core/azure-core-amqp/samples/CMakeLists.txt

```diff
@@ -10,9 +10,11 @@ cmake_minimum_required (VERSION 3.13)
 add_subdirectory ("local_server_sample")
 add_subdirectory ("local_client_sample")
 add_subdirectory ("local_client_async_sample")
-add_subdirectory ("eventhub_writer_sample")
+#add_subdirectory ("eventhub_writer_sample")
```

Comment:
why are we commenting them out?

Reply:
Because they use SASL Plain functionality which has been removed. But I don't yet have confirmation from the AMQP developers that we shouldn't support SASL Plain authentication (I think we shouldn't, but I want confirmation before I fully yank all the samples).